data/avro-schema/src/main/resources/avro/net/corda/data/membership/rpc/request/MembershipRpcRequest.avsc

```diff
@@ -16,7 +16,10 @@
         "RegistrationRpcRequest",
         "RegistrationStatusRpcRequest",
         "MGMGroupPolicyRequest",
-        "RegistrationStatusSpecificRpcRequest"
+        "RegistrationStatusSpecificRpcRequest",
+        "MutualTlsAllowCertificateRequest",
```

Comment:
Not sure if better to comment on this PR or the runtime-os PR, but this change made me wonder if there's any point to wrap this as an RPC request to the membership worker? I think it's something we can review on some of the other membership operations such as querying for registration status. For these operations we're basically trying to interact with the database so it should be possible to use the membership persistence client from the RPC worker and go direct to the database instead of going from rpc worker -> membership worker -> db worker. It seems like the only thing the membership worker is doing is checking if it's a valid holding ID which we can do anywhere.